### PR TITLE
New package: arptables-0.0.4

### PR DESCRIPTION
**Pull request by beefcurtains**
Runit service shamelessly lifted from ``ebtables``.

### CODE REVIEW

File: srcpkgs/arptables/template [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+homepage="http://ebtables.sourceforge.net/"
+distfiles="ftp://ftp.netfilter.org/pub/arptables/arptables-v0.0.4.tar.gz"
+checksum=277985e29ecd93bd759a58242cad0e02ba9d4a6e1b7795235e3b507661bc0049
+
+do_install() {
+	vbin arptables
+	vbin arptables-restore
+	vbin arptables-save
+	vman arptables.8
+	touch /etc/arptables.conf
```
Comment (xtraeme) [inline srcpkgs/arptables/template]: Either you have two options here:

- remove this line and update the runit scripts to check for this file
- add this file to `conf_files`

**Comment by thypon** [comment]
:+1:

File: srcpkgs/arptables/template (hunk 1 above)
Comment (beefcurtains) [inline srcpkgs/arptables/template]: Fixed. It was copy/pasted from ebtables, which may not have the most exemplary template.

**Comment by beefcurtains** [comment]
Travis shows commit id ``a93e0eb`` instead of ``67faa1d``, not sure how to fix this.

File: srcpkgs/arptables/files/arptables/run [hunk 2]
```diff
@@ -0,0 +1,3 @@
+#!/bin/sh
+arptables-restore < "/etc/arptables.conf"
```
Comment (xtraeme) [inline srcpkgs/arptables/files/arptables/run]: So what happens if that files doesn't exist?

Comment (beefcurtains) [inline srcpkgs/arptables/files/arptables/run]: It'll fail, of course. That's why it's created in do_install().

**Comment by beefcurtains** [comment]
Perhaps it'd be better if a configuration file could specify the location of the rules and whether saving should be skipped or not.
Then again, I could also just leave the whole runit part out. It's really just a convenience thing and I've merely included it for completeness.

**Comment by xtraeme** [comment]
I'd suggest you the following:

- Don't ship the empty `/etc/arptables.conf` conf file.
- Update the runit scripts to exit 1 if the file does not exist.

**Comment by beefcurtains** [comment]
Thanks! That makes a lot of sense. I also took the liberty of changing the default config path to ``/etc/iptables/arptables.rules`` so it's more consistent with other forks. ``ebtables`` should be updated accordingly. Eventually ``nftables`` will replace them all :smile_cat: 
For those who don't want to save the current tables on shutdown, I've added an option SKIP_SAVE that'll do just that.

Thoughts?

**Pull request merged**